docs/conf/prague/2018/news/scheduling-inclusivity.rst

```diff
@@ -0,0 +1,80 @@
+This does not occur every year, but this year it was extremely difficult to decide on dates in September, since no matter which date we chose, it would conflict with one of the holidays.
+
+When we weighed this problem against the key factors that we use to determine the dates each year, we knew that had to make a decision that is going to exclude some people from being able to attend.
+We weren't happy about this decision, but if we moved the conference to any other month, or even to the first weekend of September, we would run the risk of excluding different groups of people, or not being able to run the conference at all this year due to organizational meltdowns.
+
+What could we have done better
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aside from the risks involved with choosing different dates or canceling the Prague conference altogether, there wasn't much we could have done with regards to the scheduling this year.
+What we realized following the feedback from the community, was that we failed to communicate our decision in a transparent and empathetic way at the time of annoucning the dates.
+
+Simply put - this blog post should have been published months ago.
+
+After we announced the dates, we received and handled several inquires by email or Slack messages from community members who questioned the dates selection and asked to reschedule, but by the time we announced the dates we already had the venue contracts signed so it wasn't possible to make a change.
+
+Regardless of the actual scheduling problem, the fact that we received inquiries should have been a warning bell for us, as we saw in the past that for every person who writes to us with a question, there must be dozens more who wouldn't feel comfortable writing to us.
+We shouldn't have kept our responses to private channels, and we should have made a public accountability statement to address this much sooner.
+
+We owe it to our community to adhere to our philosophy of inclusivity in how we run our events, and if we are unable to choose the most inclusive option for our attendees, we should communicate about it promptly and transparently.
+In this case, we didn't do so, which caused portions of our community to feel as if they might have been overlooked, or worse - not welcome at our events.
+
+This was absolutely not our intention, and we are sorry that our actions had this impact on parts of our community.
+
+What will we do better in the future
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Firstly, we want to acknowledge that in 2019, the Jewish New Year holidays period begins in October, so there is absolutely no expected conflict with any dates we might choose in September.
+That said, we cannot guarantee that any future conference years (Prague, Portland, or otherwise) will not conflict with anything ever again, as this is a long-term venture and cannot predict what the far future will hold.
```

Comment:
maybe elsewhere instead of otherwise

Reply:
Maybe delete or reword 

> and cannot predict what the far future will hold

Reply:
seconded – I think I'd say, 'as this is a long-term venture, there are sure to be more difficult decisions we have to make, down the line.' or something